src/gd-frontend/src/app/gd.module.ts

```diff
@@ -0,0 +1,16 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { GdComponent } from './gd.component';
+import { GdRoutingModule } from './gd-routing.module';
+
+@NgModule({
+  imports: [
+    BrowserModule,
+    GdRoutingModule,
+  ],
+  declarations: [
+    GdComponent
+  ],
+  bootstrap: [GdComponent]
+})
+export class GdModule { }
```

Comment:
In all the examples I've seen, the top level module/component (the app entrypoint) are left named as AppModule and AppComponent, so its clear which module is at the top of the tree. It's not a hard rule though, so feel free to leave as is, just a heads up for future projects.